src/config/inter/addPSM/index.ts

```diff
@@ -0,0 +1,4 @@
+import psmJS from "./gov-start-psm.js?raw";
```

Comment:
`?raw` is a vite convention: https://vitejs.dev/guide/assets#importing-asset-as-string